app/src/src/components/modals/PickADayTimeModal.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { Platform } from 'react-native';
+import { Button, Portal, Modal } from 'react-native-paper';
+import DateTimePicker from '@react-native-community/datetimepicker';
+
+// Modal to select a day and time
+class PickADayTimeModal extends React.Component {
```

Comment:
This component looks like it can be made into a functional component. I think maybe we can quickly refactor it to be functional.